resources/user_config.ts

```diff
@@ -246,10 +246,12 @@ class UserConfig {
       // System Language
       if (e.detail.systemLocale) {
         options.SystemLocale = e.detail.systemLocale;
-        options.ShortLocale = e.detail.systemLocale.substring(0, 2);
-        if (options.ShortLocale === 'zh')
-          options.ShortLocale = 'cn';
-        if (!supportedLanguage.includes(options.ShortLocale))
+        let shortLocale = e.detail.systemLocale.substring(0, 2);
+        if (shortLocale === 'zh')
+          shortLocale = 'cn';
+        if (isLang(shortLocale))
+          options.ShortLocale = shortLocale;
```

Comment:
Ah, this is a nice type improvement! Thanks for the followup PR.

Is it possible to do the same below for `options.DipslayLanguage` and remove the local `supportedLanguage` array?

Reply:
Done 😃